pkg/cluster/nodeutils/util.go

```diff
@@ -82,7 +83,9 @@ func LoadImageArchive(n nodes.Node, image io.Reader) error {
 	if err != nil {
 		return err
 	}
-	cmd := n.Command("ctr", "--namespace=k8s.io", "images", "import", "--all-platforms", "--digests", "--snapshotter="+snapshotter, "-").SetStdin(image)
+	// --local=true: Disables the Transfer API mode that is enabled by default since containerd v2.0.
+	// Workaround for `ctr: rpc error: code = InvalidArgument desc = unable to initialize unpacker: no unpack platforms defined: invalid argument`
```

Comment:
We probably need a long term solution for this?

Reply:
Also: https://github.com/kubernetes-sigs/kind/pull/3805

Reply:
Opened an issue:
- https://github.com/containerd/containerd/issues/11228

Reply:
PR
- https://github.com/containerd/containerd/pull/11229

Reply:
thank you!

Reply:
Updated to v2.0.2